zutil.c

```diff
@@ -22,16 +22,7 @@ const char * const zng_errmsg[10] = {
     (const char *)""
 };
 
-const char zlibng_string[] =
```

Comment:
String not used for anything..

Reply:
We should expose this string to external applications somehow, don't you think?

Reply:
I suppose. I'm not sure the usefulness of it or the copyright strings, but it is what it is..

Reply:
I have added back the string.